README.md

```diff
@@ -165,8 +165,8 @@ declare const headlessCheckout: {
 | Payment Methods       | psdk-payment-methods | ✅         |
 | Saved Methods         | ❔                   | 🕑         |
 | Payment Form Messages | ❔                   | 🕑         |
-| Checkbox              | ❔                   | 🕑         |
-| Select                | ❔                   | 🕑         |
+| Checkbox              | psdk-checkbox        | ✅         |
+| Select                | psdk-select          | ✅         |
 | Apple Pay Button      | ❔                   | 🕑         |
 | Google Pay Button     | ❔                   | 🕑         |
 | Delete Account Button | ❔                   | 🕑         |
```

Comment:
we also have receipt component

Reply:
We discussed this issue and the Receipt component is obsolete. FinanceDetails is the actual one. It's already documented.